ui/src/main/java/be/appwise/ui/Buttons.kt

```diff
@@ -0,0 +1,179 @@
+package be.appwise.ui
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowRight
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.LocalTextStyle
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+object Buttons {
+    @Composable
+    fun Primary(
+        text: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        contentColor: Color = MaterialTheme.colorScheme.onPrimary,
+        containerColor: Color = MaterialTheme.colorScheme.primary,
+        style: TextStyle = LocalTextStyle.current,
+        onClick: () -> Unit = {}
+    ) {
+        Button(
+            onClick = onClick,
+            enabled = enabled,
+            modifier = modifier,
+            shape = RoundedCornerShape(5.dp),
```

Comment:
I would move the shape to the parameters of the function as it is something that we will have to overwrite quite a lot for different designs and projects